### DISCUSSION TITLE
Support i18n

### DISCUSSION THREAD

**Discussion by Kenzo-Wada** [vikejs/vike#1312] [Discussion] [answered]
Hi,

Are you considerring about supporting i18n?
if so, since Im a native Japanese speaker, I'd love to create Japanese document like `ja.vike.dev` at first if its ok.

Also, I thought that chinese site is old so that it should be archived or updated.

https://vike.dev/languages

**Comment by usk94** [comment]
@Kenzo-Wada 
Hi, Kenzo! I'm Japanese as well, and if you don't mind, I'd like to do some documentation work with you too!😀

**Reply by usk94** [reply]
I recently started contributing to vike.js, and wanted to write a Japanese version of the documentation for the v1 design.
Sorry to interrupt, If you want to do it alone, feel free to let me know👌

**Reply by Kenzo-Wada** [reply]
Hi, @usk94 !

I'd appreciate if you'll work this with me! If core member of vike agree with this suggestion, lets do it together! 👌

**Reply by brillout** [reply, marked as answer]
A Japanese translation would be neat indeed 👍 I'd suggest the same we did for the Chinese translation, namely to copy the `docs/` directory into in a new repository.

**Reply by Kenzo-Wada** [reply]
Thank you @brillout !

I've made repo for this project: https://github.com/Kenzo-Wada/vike-docs-ja
I'll let you know when it's all done and ready for deployed!

**Reply by Kenzo-Wada** [reply]
@usk94 

I'll welcome your contribution. Be free for asking me help if anything!
(Also, lets speek in japanese if you dont mind. it's more smooth for me lol)

**Reply by usk94** [reply]
@Kenzo-Wada 

Thanks for creating the repository!😆
Ok, let's communicate in Japanese during the documentation process.
(たのしみです、よろしくお願いします😆)

**Reply by Kenzo-Wada** [reply]
@brillout 

As we both discussed efficient ways to sync changes to the vike.dev docs, we came to the conclusion that it would be better to create a folked repository instead of creating a new repository. What do you think?

The reason is that if we start with a completely separate repository, we would have to manually catch and fix changes to the vike.dev docs, which would take a lot of man-hours and would likely result in leaks.
On the other hand, I'm concerned that if we folk, we'll be importing the contents of directories other than docs that we don't need.

**Reply by brillout** [reply]
👍 I guess forking is fine as well.

**Reply by Kenzo-Wada** [reply]
thankyou!

**Reply by Kenzo-Wada** [reply]
@usk94 

i've updated repo!
https://github.com/vike-ja-support/vike-docs-ja

**Reply by lourot** [reply]
Thanks a lot for doing that @Kenzo-Wada! Once you have set up GitHub Pages on your repo we can create a DNS entry ja.vike.dev to point at it :pray:

**Reply by Kenzo-Wada** [reply]
@AurelienLourot 

Thank you for your support!
One question, is there any problem with us if I deploy  using Vercel?(its much more easy to deploy production and also pr preview)

**Reply by lourot** [reply]
Fine with us :)

**Reply by JustFly1984** [reply]
I have multiple projects in vite + remix, using i18n with path prefixes /jp/, /fr/ etc. Is there support for i18n with vike? Would love to move some of my projects from remix SSR, which uses cloudflare pages functions, to vike SSG, to drop SSR, and save on expenses.

**Reply by brillout** [reply]
@JustFly1984 https://vike.dev/i18n
